Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -198,72 +211,106 @@ fun HomeContent(
-                        color = MaterialTheme.colors.primary.copy(alpha = 0.38f),
-                        startYPercentage = 1f,
-                        endYPercentage = 0f
-                    )
-            ) {
-                // Draw a scrim over the status bar which matches the app bar
-                Spacer(
-                    Modifier
-                        .background(appBarColor)
-                        .fillMaxWidth()
-                        .windowInsetsTopHeight(WindowInsets.statusBars)
-                )
+            val scrimColor = MaterialTheme.colors.primary.copy(alpha = 0.38f)
+            Scaffold(
+                topBar = {
+                    Column(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .background(color = scrimColor)
+                    ) {
+                        // Draw a scrim over the status bar which matches the app bar
+                        Spacer(
+                            Modifier
+                                .background(appBarColor)
+                                .fillMaxWidth()
+                                .windowInsetsTopHeight(WindowInsets.statusBars)
+                        )
+                        HomeAppBar(
+                            backgroundColor = appBarColor,
+                            modifier = Modifier.fillMaxWidth()
+                        )
+                    }
+                },
+            ) { contentPadding ->
+                LazyColumn(
+                    contentPadding = contentPadding,
+                    modifier = Modifier.fillMaxSize()
+                ) {
+                    if (featuredPodcasts.isNotEmpty()) {
+                        item {
+                            FollowedPodcastItem(
+                                items = featuredPodcasts,
+                                pagerState = pagerState,
+                                onPodcastUnfollowed = onPodcastUnfollowed,
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .verticalGradientScrim(
+                                        color = scrimColor,
+                                        startYPercentage = 1f,
+                                        endYPercentage = 0f
+                                    )
+                            )
+                        }
 
-                HomeAppBar(
-                    backgroundColor = appBarColor,
-                    modifier = Modifier.fillMaxWidth()
-                )
+                        if (isRefreshing) {
+                            // TODO show a progress indicator or similar
+                        }
 
-                if (featuredPodcasts.isNotEmpty()) {
-                    Spacer(Modifier.height(16.dp))
+                        stickyHeader {
+                            if (homeCategories.isNotEmpty()) {
```

Comment:
nit: It should be the other way around I think?

```
if (homeCategories.isNotEmpty()) {
   stickyHeader {}
}
```

Reply:
Good catch. Fixed it